src/test/java/hudson/plugins/throttleconcurrents/ThrottleConcurrentTest.java

```diff
@@ -25,6 +26,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Ignore("Depends on a newer version of Guava than can be used with Pipeline")
```

Comment:
I take that back - did some experimenting and there is something weird once you've got enough Pipeline tasks in queue. Working on it.